java/generate-offline-map-with-local-basemap/README.md

```diff
@@ -20,28 +20,31 @@ The author of a web map can support the use of basemaps which are already on a d
 
 ## How to use the sample
 
-1. Click on the "Generate offline map" button.
-1. You will be prompted to choose whether you wish to use the "naperville_imagery.tpk" basemap which is already on the device or download the online basemap.
-1. If you choose to download the online basemap, the offline map will be generated with the same (topographic) basemap as the online web map. To download the Esri basemap, you may be prompted to sign in to ArcGIS.com.
-1. If you choose to use the basemap from the device, the offline map will be generated with the local imagery basemap. The download will be quicker since no tiles are exported or downloaded. Since the application is not exporting online ArcGIS Online basemaps you will not need to log-in.
+1. Tap on "Generate Offline Map".
+2. You will be prompted to choose whether you wish to download the online basemap or use the "naperville_imagery.tpk" basemap which is already on the device.
+3. If you choose to download the online basemap, the offline map will be generated with the same (topographic) basemap as the online web map.
+4. To download the Esri basemap, you may be prompted to sign in to ArcGIS.com.
+5. If you choose to use the basemap from the device, the offline map will be generated with the local imagery basemap. The download will be quicker since no tiles are exported or downloaded.
+6. Since the application is not exporting online ArcGIS Online basemaps you will not need to log-in.
 
 ## How it works
 
-1. Create an `ArcGISMap` with a portal item pointing to the web map.
-1. Create `GenerateOfflineMapParameters` specifying the download area geometry, min scale, and max scale.
-1. Once the generate offline map parameters are created, check the `getReferenceBasemapFilename()` property. The author of an online web map can configure this setting to indicate the name of a suitable basemap. In this sample, the app checks the local device for the suggested "naperville_imagery.tpk" file.
-1. If the user chooses to use the basemap on the device, use `setReferenceBasemapDirectory()` on the generate offline map parameters to set the absolute path of the directory which contains the .tpk file.
-1. Create an `OfflineMapTask` the generate offline map parameters.
-1. Create the offline map job and start it.
-1. When the job is done, use `getOfflineMap` on the `GenerateOfflineMapResult` to get the map.
+The sample creates a `PortalItem` object using a web map's ID. This portal item is used to initialize an `OfflineMapTask` object. When the button is tapped, the sample requests the default parameters for the task, with the selected extent, by calling `OfflineMapTask.createDefaultGenerateOfflineMapParameters`. 
```

Comment:
Looks like there was an oversight when the design was updated here in common-samples, in that the numbering of the sections have been removed. For "How it works", we should always use a numbered list.

Reply:
This design also breaks standard a little by being more verbose and including use of user interaction amongst API calls. Ideally these would be removed just so as to focus on the steps the API is using. Would you be comfortable having a go at this to make this section a little more standardized like our other How it works sections?

There may be parts of this section which would be useful to maintain in either the "Additional info" section or as code comments within the sample itself.